package.json

```diff
@@ -59,9 +63,9 @@
   "scripts": {
     "build-css": "sass --no-source-map src/index.scss src/index.css",
     "watch-css": "npm run build-css && sass --no-source-map -w src/index.scss src/index.css",
-    "start-js": "react-scripts start",
+    "start-js": "react-scripts --openssl-legacy-provider start",
```

Comment:
I think we should map which dependencies need this `--openssl-legacy-provider`. So we know when we can remove this flag.

Reply:
As described on the updated PR, the most relevant dependency for removing this flag is being discussed on #441